legal-api/requirements.txt

```diff
@@ -30,7 +30,7 @@ ecdsa==0.14.1
 expiringdict==1.1.4
 flask-jwt-oidc==0.3.0
 flask-restx==0.3.0
-google-auth==2.16.2
+google-auth==2.29.0
```

Comment:
Saw conflicts on version of google-auth, so update it as per the version used by gcp-queue https://github.com/bcgov/sbc-connect-common/blob/main/python/gcp-queue/poetry.lock#L379